src/public_key.rs

```diff
@@ -24,13 +24,30 @@ pub trait PublicKeySize {
 /// network.
 ///
 /// Public keys can convert to and from their binary and base58 representation
-#[derive(Debug, Clone, PartialEq, Hash)]
+#[derive(Clone, PartialEq, Hash)]
 pub struct PublicKey {
     /// The network this public key is valid for
     pub network: Network,
     inner: PublicKeyRepr,
 }
 
+impl fmt::Debug for PublicKey {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        let r#type = match self.inner {
+            PublicKeyRepr::Ed25519(_) => "ed25519",
+            PublicKeyRepr::EccCompact(_) => "ecc_compact",
+        };
+        let output = format!(
+            "PublicKey {{ \
+        network: {:?}, type: {}, base58: {} }}",
```

Comment:
```suggestion
        network: {:?}, type: {}, addresss: {} }}",
```

Reply:
Resolved in commit c1693cabc241f2eef2611ff654b1a53b3619957a.